### ISSUE TITLE
Forum Report: Learning lasso from disassembling?

### ISSUE THREAD

**Issue by JustinLove** [jasonrohrer/OneLifeData7#513]
> It feels like a bug to use up a tool slot just by breaking a lasso back down into two ropes.
> 
> There shouldn't be any lassoing skill needed to cut a long rope back into shorter lengths.

https://onehouronelife.com/forums/viewtopic.php?id=8534

**Comment by jasonrohrer** [comment]
Yeah... not sure about this.  It's tough to tease these things apart in the automatic implementation...

**Comment by ghost** [comment]
> Yeah... not sure about this. It's tough to tease these things apart in the automatic implementation...

idea:
Marking “stakes” and “auto-orienting wall stakes” as “+toolConstruction”.
change so that if both target and newTarget were +tool, requires skill.

**Comment by Tarr-OHOL** [comment]
Same thing for removing the flint tip off a firebow drill. Doing so learns the skill.

**Comment by ghost** [comment]
Deconstructing knife also learns skill
